src/ReleaseHistory.md

```diff
@@ -29,4 +29,8 @@
 
 ## **0.63.0** [Pointer](https://www.nuget.org/packages/Microsoft.Json.Pointer/0.63.0) | [Schema](https://www.nuget.org/packages/Microsoft.Json.Schema/0.63.0)| [Schema.ToDotNet](https://www.nuget.org/packages/Microsoft.Json.Schema.ToDotNet/0.63.0)| [Schema.Validation](https://www.nuget.org/packages/Microsoft.Json.Schema.Validation/0.63.0)
 
-* Update SARIF dependency to private build which is effectively 2.0.0-csd.2.beta.2019-02-20 (the core SARIF eballots).
\ No newline at end of file
+* Update SARIF dependency to private build which is effectively 2.0.0-csd.2.beta.2019-02-20 (the core SARIF eballots).
+
+## **1.0.0** [Pointer](https://www.nuget.org/packages/Microsoft.Json.Pointer/1.0.0) | [Schema](https://www.nuget.org/packages/Microsoft.Json.Schema/1.0.0)| [Schema.ToDotNet](https://www.nuget.org/packages/Microsoft.Json.Schema.ToDotNet/1.0.0)| [Schema.Validation](https://www.nuget.org/packages/Microsoft.Json.Schema.Validation/1.0.0)
```

Comment:
>*1.0.0* [](start = 4, length = 7)

wait did you actually reversion your package yet? not yet, i guess, because you are introducing a dependency on a prerelease SARIF SDK. :) you'll need to update yet again after Harlene checks in to make this all happen. maybe we should wait until she does that (and you can make the 'virtual method' code emit change now instead.